tests/virtual_machines_test.go

```diff
@@ -124,19 +125,44 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			})
 		})
 
-		Context("When the client wants to create a vm on an opted-in namespace", func() {
-			It("should create a vm object and automatically assign a static MAC address", func() {
+		Context("When the client creates a vm on an opted-in namespace", func() {
+			var (
+				vm *kubevirtv1.VirtualMachine
+			)
+			BeforeEach(func() {
 				err := initKubemacpoolParams(rangeStart, rangeEnd)
 				Expect(err).ToNot(HaveOccurred())
 
-				vm := CreateVmObject(TestNamespace, false, []kubevirtv1.Interface{newInterface("br", "")},
+				vm = CreateVmObject(TestNamespace, false, []kubevirtv1.Interface{newInterface("br", "")},
 					[]kubevirtv1.Network{newNetwork("br")})
 
 				err = testClient.VirtClient.Create(context.TODO(), vm)
 				Expect(err).ToNot(HaveOccurred())
+			})
+			It("should has the vm with an automatically assign static MAC address within range", func() {
```

Comment:
please rephrase
`should automatically assign the vm with static MAC address within range`